rcl/test/rcl/test_publisher.cpp

```diff
@@ -21,11 +21,15 @@
 #include "test_msgs/msg/strings.h"
 #include "rosidl_runtime_c/string_functions.h"
 
-#include "./failing_allocator_functions.hpp"
+#include "mimick/mimick.h"
```

Comment:
@Blast545 nit: unused (at least directly)?

Reply:
It's used, see [Line 331](https://github.com/ros2/rcl/blob/e0fb32d51ce69f92fe6b272e2838c33b09e7c344/rcl/test/rcl/test_publisher.cpp#L331).